src/worker/generate-kmp-json.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  KmpJsonFile,
+  KmpJsonFileSystem,
+  KmpJsonFileOptions,
+} from "@common/kmp-json-file";
+
+/**
+ * Given languages, generate a .kmp json file which contains metadata for .kmp zip archive.
+ */
+export function generateKmpJson(
+  keymanDeveloperVersion: string,
+  fileVersion: string,
+  followKeyboardVersion: boolean
+): string {
+  const system: KmpJsonFileSystem = { keymanDeveloperVersion, fileVersion };
+  const options: KmpJsonFileOptions = { followKeyboardVersion };
+  const kmpJson: KmpJsonFile = { system, options };
+
+  return JSON.stringify(kmpJson);
+}
```

Comment:
Honestly, I don't really find the value in this function, especially since it doesn't allow you to specify the lexical model — which is the most important part for our purposes!

I won't request changes for this PR, but if I did, I'd suggest just omitting this function and its test! 😂